packages/@angular/cli/tasks/schematic-get-options.ts

```diff
@@ -19,10 +18,7 @@ export interface SchematicAvailableOptions {
 
 export default Task.extend({
   run: function (options: SchematicGetOptions): Promise<SchematicAvailableOptions[]> {
-    const collectionName = options.collectionName ||
-      CliConfig.getValue('defaults.schematics.collection');
```

Comment:
No need to retrieve config value from multiple places. Now it's always responsibility of the  caller.